mkdocs.yml

```diff
@@ -90,7 +90,8 @@ pages:
    - '____ CDN Source': 'adapters/stochastic_cdn.md'
    - '____ Logs Source': 'adapters/stochastic_logs.md'
    - '____ Search Cluster Source': 'adapters/stochastic_search_cluster.md'
-   
+   - '____ Adapter Implementation API': 'adapters/adapter_api.md'
```

Comment:
This would hide the doc from the left pane. How should it be findable by developers?